config/val.config.src

```diff
@@ -82,6 +84,11 @@
                       header_table_size => 4096,
                       enable_push => 1,
                       max_concurrent_streams => unlimited,
```

Comment:
do we want to change things as well?

Reply:
we could but i doubt it is causing any probs...its the max num of streams per connection